lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -139,17 +139,24 @@ class ElementScreenshotRenderer {
    * @param {LH.Artifacts.FullPageScreenshot} fullPageScreenshot
    */
   static installOverlayFeature(dom, templateContext, fullPageScreenshot) {
-    const reportEl = dom.find('.lh-report', dom.document());
-    const screenshotOverlayClass = 'lh-feature-screenshot-overlay';
-    if (reportEl.classList.contains(screenshotOverlayClass)) return;
-    reportEl.classList.add(screenshotOverlayClass);
+    const topbarEl = dom.find('.lh-topbar', dom.document());
+    const containerEl = topbarEl.parentElement;
```

Comment:
ah so this *should* be the `.lh-report` in the standalone report, but might be different in DevTools/PSI?

Reply:
nope. `lh-report` is insider `lh-container`.

standalone report:

![image](https://user-images.githubusercontent.com/4071474/102416513-966d9700-3fbf-11eb-83e5-61a0e906f20f.png)


CDT:

![image](https://user-images.githubusercontent.com/4071474/102416331-3ecf2b80-3fbf-11eb-969e-f13b4265c798.png)
![image](https://user-images.githubusercontent.com/4071474/102416433-6a521600-3fbf-11eb-8832-0679c274caa3.png)